js-exercises/my-promise/myPromise.test.js

```diff
@@ -0,0 +1,7 @@
+import { myPromise } from './myPromise';
+
+describe('Template Test', () => {
+  test('Template Test', () => {
+    expect(true).toBe('tests for myPromise not written');
+  });
+});
```

Comment:
I would like you to add more tests here, I am sure you can find and write some good one